scripts/prepare-machine.ps1

```diff
@@ -537,7 +537,12 @@ if ($IsLinux) {
         sudo apt-get install -y cmake
         sudo apt-get install -y build-essential
         sudo apt-get install -y liblttng-ust-dev
-        sudo apt-get install -y babeltrace
+        # Try to install babeltrace2 first, then fallback to babeltrace
+        try {
+            sudo apt-get install -y babeltrace2 2>&1 | Out-Null
```

Comment:
I don't see why this would throw on failure, this won't allow to fallback to babeltrace if the installation of babeltrace2 fails.